wiggum_app/controller.py

```diff
@@ -248,20 +252,24 @@ def main():
 
         # initial for visualize.html page
         if action == 'page_load':
+            # check if page is reload or not
+            if redirect_flag:
```

Comment:
What is this for? Can you add comments that describe ti in a little more detail for the sake of future maintainability

Reply:
Added